eventActions/mentionSpam.js

```diff
@@ -0,0 +1,24 @@
+const config = require('../config.json')
+
+class mentionSpam {
+    static async preventMentionSpam(message) {
+        if (message.mentions.members.size >= 10) {
```

Comment:
Maybe the threshold should be a configurable variable instead of a seemingly constant variable?

Reply:
Good idea. I'll make this a variable in config.json.example.